utils/utils.js

```
@@ -0,0 +1,33 @@
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+//var expect = chai.expect;
+
+module.exports.waitNotPresent = function(selector) {
```

In this case dont care about use dynamic timouts. **It should fail fast**. Otherwise we have performance issues or wrong configuration. And by model user can wait maximum 2secs for element to appear. Here its 5x times more